libs/core/src/components/pds-layout-row/pds-layout-row.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, Prop, h, Host} from '@stencil/core';
+
+import { BoxTShirtSizeType } from '../../utils/types';
+
+@Component({
+  tag: 'pds-layout-row',
+  styleUrl: 'pds-layout-row.scss',
+})
+export class PdsLayoutRow {
```

Comment:
we should just call it `PdsRow`. Let's put it under a sub-folder under components. e.g `components/layout/pds-row`. But I strongly feel it would add an extra layer of confusion.